mpa/modules/hooks/auxiliary_hooks.py

```diff
@@ -106,91 +102,49 @@ def func(feature_map: Union[torch.Tensor, list[torch.Tensor]], _fpn_idx: int = 0
             feature_map = feature_map[_fpn_idx]
```

Comment:
How about add `assert` or `raise` if `fpn_idx` is out of scope?
i.e) When the FPN has [0,1,2] indices, if user input the variable `fpn_idx` as 3, we need to description.

Reply:
It will raise exception here anyway. Do we need to do this unless we are taking care of the exception?

Reply:
No, we don't need it. Just want to give a kind description. i.e. by checking the error message, users easily find the real FPN range.